lessons/en/basics/date_time.md

```diff
@@ -42,7 +42,7 @@ But there is a catch: as you may have noticed, this struct only contains the tim
 
 ## Date
 
-Contrary to `Time`, a `Date` struct has info about the current date, without any info about current time.
```

Comment:
Word "current" may be a bit misleading in this case, because `Date` may contain _any_ date, not only the current.